from timeit import timeit



# try:
#     with open('006_exceptions.py') as file:
#         print('File opened.')
        # file.__exit__() #content management protocol __enter__
#     # file = open('006_exceptions.py')
#     age = int(input('Age: '))
#     xfactor = 10 / age
# except (ValueError, ZeroDivisionError ) as e:
#     print('Enter valid age')
#     print(e)

# else:
#     print('NO exceptions thrown')  
# finally: # is always executed wheather we have exceptions or not  
#     file.close()


def calculate_xfactor(age):
    pass
